<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EVE Ratting Tools</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>High Tower Text</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>EVE Ratting Tools</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Ratting Count</string>
    </property>
    <widget class="QPushButton" name="btn_start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stop">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>20</width>
       <height>121</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="static_rattingTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ratting Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="static_totalIncome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Income</string>
     </property>
    </widget>
    <widget class="QLabel" name="static_ESSIncome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ESS Income</string>
     </property>
    </widget>
    <widget class="QLabel" name="static_ISKSpeed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ISK Speed</string>
     </property>
    </widget>
    <widget class="QLabel" name="active_rattingTime">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="active_totalIncome">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="active_ESSIncome">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="active_ISKSpeed">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="static_todayIncome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TodayIncome</string>
     </property>
    </widget>
    <widget class="QLabel" name="active_todayIncome">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="static_dailyGoals">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DailyGoals</string>
     </property>
    </widget>
    <widget class="QLabel" name="active_dailyGoals">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>391</width>
      <height>191</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Graphic Command</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSettins"/>
    <addaction name="actionResearch_active_log"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionResearch_active_log">
   <property name="text">
    <string>Rescan active log</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettins">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startCount()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_stop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopCount()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettins</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startCount()</slot>
  <slot>stopCount()</slot>
  <slot>showAbout()</slot>
  <slot>showSettings()</slot>
 </slots>
</ui>
